src/main/java/com/opentable/kafka/builders/KafkaBuilderConfiguration.java

```diff
@@ -25,8 +25,7 @@
  */
 @Configuration
 @Import({
-        KafkaConsumerBuilderFactoryBean.class,
-        KafkaProducerBuilderFactoryBean.class,
+        KafkaBuilderFactoryBean.class,
```

Comment:
The annotation now must be renamed. It's singular, not plural now.

Reply:
done.